src/hooks/useSignInDialog/Steps/AccountStep.style.ts

```diff
@@ -0,0 +1,116 @@
+import { Spinner, Button, Text, RadioButton } from '@/shared/components'
+import { sizes, colors, typography, transitions } from '@/shared/theme'
+import styled from '@emotion/styled'
+
+type AccountWrapperProps = {
+  isSelected?: boolean
+}
+
+export const IconGroup = styled.div`
+  display: flex;
+  align-items: center;
+  img,
+  svg {
+    display: block;
+    margin: 0 ${sizes(2)};
+  }
```

Comment:
Couldn't this be just selected with `> *`? I'd like to avoid selecting by explicit element name, I wouldn't want our styles in the whole app to break if we stop inlining SVGs for example